lib/facter/confluence_version.rb

```diff
@@ -1,11 +1,12 @@
 Facter.add(:confluence_version) do
   setcode do
     ps = Facter::Util::Resolution.exec('ps ax')
-    confluence_process = ps && ps.split("\n").find { |x| x.include?('atlassian-confluence') }
-    if confluence_process.nil?
+    confluence_process = ps && ps.split("\n").find { |x| x.include?('atlassian-confluence-') }
```

Comment:
Changed to pgrep now; also only running ps once so should be up to twice as fast!